docs/experimental/examples/endpoints_and_collections/create_guest_collection/index.rst

```diff
@@ -2,8 +2,60 @@
 Creating a Guest Collection
 ===========================
 
-TODO
+Within the Globus Ecosystem, data is managed through the abstraction of *Collections*.
+The example included on this page demonstrate how to create specifically a
+*Guest Collection* using the Globus Python SDK. Guest collections, formerly known as
+"Shares", are collections which provide access to a subdirectory of an existing
+collection through a particular user or client's local permissions.
 
-.. literalinclude:: user_guest_collection.py
-    :caption: ``user_guest_collection.py`` [:download:`download <user_guest_collection.py>`]
-    :language: python
+Guest collections are a great way to set up data automation. They may be scoped down
+to a particular directory within an existing "Mapped Collection" and don't implicitly
+inherit the same authorization timeout requirements as their parent Mapped Collection.
+Once created, they can be shared to other users/entities, in effect giving another
+entity access, through you, to some underlying data.
+
+.. Note::
```

Comment:
I think this may warrant something more attention grabbing than a note. What do you think about it being a `warning` instead? 

Warnings in docs need to be used sparingly for good flow, but I think this is enough of a "gotcha" that it's appropriate.

Reply:
Sure,  can update to warning.